content/components/core/code/scss-variables.md

```diff
@@ -0,0 +1,71 @@
+---
+layout: section
+---
+
+## SCSS Variables
+
+### Default Theme Colours
+
+| Variable Name               | Default Value | Description                                                                                                |
+|-----------------------------|---------------|------------------------------------------------------------------------------------------------------------|
+| `$AU-color-foreground-text`   | `#313131`       | Colour used for paragraphs, headings etc.                                                                                 |
+| `$AU-color-foreground-action` | `#00698F`       | Colour of links, buttons or other action items.                                                             |
+| `$AU-color-foreground-focus`  | `#9263DE`       | Color around an element that is in focus state. |
+| `$AU-color-background`        | `#FFFFFF`       | Background colour.                                                                            |
+
+
+### Dark Theme Colours
+
+| Variable Name               | Default Value | Description                                                                                                |
+|-----------------------------|---------------|------------------------------------------------------------------------------------------------------------|
+| `$AU-colordark-foreground-text`   | `#FFFFFF`       | Colour used for paragraphs, headings etc.                                                                                  |
+| `$AU-colordark-foreground-action` | `#61DAFF`       | Colour of links, buttons or other action items.                                                             |
+| `$AU-colordark-foreground-focus`  | `#C390F9`       | Color around an element that is in focus state. |
```

Comment:
I've noticed a few times `Color` and `Colour` have been used. We use `color` in our code and `colour` in our documentation.